src/components/TimerLabel/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTimer } from '../../utility/hooks/useTimer';
+import { SpanTimer } from './styles';
+
+const TimerLabel = ({ date, size, color, showlabel }) => {
+  let { timeLeft } = useTimer(date);
+  const timerComponents = [];
+
+  Object.keys(timeLeft).forEach((interval) => {
+    let label = showlabel
+      ? ` ${timeLeft[interval]} ${interval.toLocaleUpperCase()} `
```

Comment:
Separate the number from the label. Put each one inside a `<div>` so it would be easier to format its visuals like this:

<img width="245" alt="Captura de Pantalla 2020-08-26 a la(s) 7 51 11 a  m" src="https://user-images.githubusercontent.com/17033709/91305640-ed313580-e770-11ea-8160-0c87ffda6f25.png">

Reply:
OK
<img width="837" alt="Captura de Pantalla 2020-08-26 a la(s) 15 21 36" src="https://user-images.githubusercontent.com/5388080/91353088-63a15800-e7b0-11ea-8234-4253cc8697f0.png">